### PR TITLE
Devise backend Set Up

### PR DESCRIPTION
**Pull request by ridwvn22**
### Changes
Give a short summary of the changes made in this PR. 

creadted/modified authentication controllers, added migration to create association for user/educators, created a seed user and modified the educator seed, ran rails db:seed & tested.

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1206660705748710/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

### Before you submit for review, Did you...

- [X] Added relevant tests (if applicable)
- [X] Linked Asana ticket to PR
- [ ] Added comments, notes, or questions for reviewwer
- [ ] Added relevant screenshots
- [X] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-        <%= yield %>
-        <%= render 'layouts/sidebar_nav' %>
-      </div>
-    </main>
-  </body>
+<body>
+  <main class="container mx-auto mt-28 px-5 flex">
+    <!--This is where you will add render lines to render the partials-->
+    <% if current_user %>
+      <%= render 'layouts/sidebar_nav' %>
```
Comment (camckin10) [inline app/views/layouts/application.html.erb:16]: You can the extra 'render layouts/sidebar_nav' statement from this file. 
The sidebar nav is already being rendered further down in the code.

File: app/models/educator.rb [hunk 2]
```diff
@@ -4,4 +4,5 @@ class Educator < ApplicationRecord
   belongs_to :school
   has_and_belongs_to_many :classrooms
   has_many :students, through: :classrooms
+  belongs_to :user
```
Comment (edithemmings) [inline app/models/educator.rb:7]: The educator model already has some tests that verify the existing associations. Do you see where you could add some test coverage for this new user association?

**Review by edithemmings — APPROVED** [review]
Looks great! Nice work!

**Review by edithemmings — APPROVED** [review]
Looks great! Nice work!

**Pull request merged**